network-api/networkapi/settings.py

```diff
@@ -269,6 +269,7 @@
             ])),
             'libraries': {
                 'bg_nav_tags': 'networkapi.buyersguide.templatetags.bg_nav_tags',
+                'blog_tags': 'networkapi.wagtailpages.templatetags.blog_tags',
```

Comment:
...will never forget to register tem

Reply:
we may have to revisit whether this deserves its own file or whether it makes sense to fold it in somewhere else... (it's technically zero code related to blogs)